app/models/IvdSubmission.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import pages._
+import play.api.libs.json.{Json, Reads, Writes}
+
+import java.time.LocalDate
+
+case class IvdSubmission(userType: UserType,
+                         numEntries: NumberOfEntries,
+                         acceptedBeforeBrexit: Boolean,
+                         additionalInfo: String = "Not Applicable",
+                         entryDetails: EntryDetails,
+                         originalCpc: String,
+                         declarantContactDetails: ContactDetails,
+                         declarantAddress: ContactAddress,
+                         defermentType: Option[String] = None,
+                         defermentAccountNumber: Option[String] = None,
+                         additionalDefermentNumber: Option[String] = None,
+                         amendedItems: Seq[UnderpaymentReason] = Seq.empty,
+                         underpaymentDetails: Seq[UnderpaymentDetail] = Seq.empty,
+                         documentsSupplied: Seq[String] = Seq.empty,
+                         supportingDocuments: Seq[FileUploadInfo] = Seq.empty)
+
+object IvdSubmission {
+  implicit val writes: Writes[IvdSubmission] = (data: IvdSubmission) => {
+
+    val brexit = LocalDate.parse("2021-01-01")
```

Comment:
We have this date in FixedConfig trait. Should we use it from there or refactor to remove that?